packages/server/lambdas/alfresco/create-folder.ts

```diff
@@ -19,12 +19,15 @@ let fileEndpointsCache: Array<CategoryDataBase> = [];
 const postFolder = async (options: RequestInit, nodeId: string): Promise<AlfrescoResponse | undefined> => {
   const alfrescoCoreAPIUrl = `${getAlfrescoUrlBase()}/alfresco/versions/1`;
   const url = `${alfrescoCoreAPIUrl}/nodes/${nodeId}/children`;
-  try {
-    const res = await fetch(url, options);
-    const result = (await res.json()) as AlfrescoResponse;
-    return result;
-  } catch (err) {
-    console.log(err);
+  const res = await fetch(url, options);
+  if (res.ok) {
+    const text = await res.text();
+    if (!text) return;
+    const result = JSON.parse(text);
+    return result as AlfrescoResponse;
+  } else {
+    console.error('HTTP error:', res.status, res.statusText);
+    throw new Error(`HTTP error: ${res.status} ${res.statusText}`);
```

Comment:
Is it possible you could move this line to a new helper function (e.g. alfrescoFetch, apiFetch or something like that?) in util and re-use it?

```
export const alfrescoFetch = (url, options) => ... 
```